### PR TITLE
Add basic HTML and CSS portion for idea grid section

### PR DESCRIPTION
**Pull request by edwardkrupicka**
What (if any) features are you implementing?
- Added gid section in HTML file and basic CSS styling for that grid section.
What (if anything) did you refactor?
- No.
Were there any issues that arose?
-None, but it's going to require more CSS styling to make it look better.
Is there anything that you need from your teammate?
- Please review, and suggestions would be helpful.
Any other comments, questions, or concerns?

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -34,6 +43,4 @@ <h2>Body</h2>
 
 
 
-      </main>
```
Comment (ivonne-hernandez) [inline index.html:37]: We may want to move these tags up.

File: styles.css [hunk 2]
```diff
@@ -30,3 +30,17 @@
   width: 10px;
   height: 10px;
 }
+
+.idea-grid {
+  background-color: EAEAF4
```
Comment (ivonne-hernandez) [inline styles.css:35]: We'll need a semi-colon here.

File: styles.css [hunk 3]
```diff
@@ -30,3 +30,17 @@
   width: 10px;
   height: 10px;
 }
+
+.idea-grid {
```
Comment (reganlosey) [inline styles.css:34]: This must have taken hours. Good stuff, we'll come back to this later.

File: styles.css [hunk 4]
```diff
@@ -30,3 +30,17 @@
...
+  background-color: EAEAF4
+  display: grid;
+  grid-template-columns: 1fr 7fr;
+
```
Comment (ivonne-hernandez) [inline styles.css:38]: If this space isn't needed, we can delete it.

**Review by reganlosey — COMMENTED** [review]
This must have taken hours. Good stuff, we'll come back to this later.

**Review by ivonne-hernandez — APPROVED** [review]
Let's make sure we refactor a lot of these tags and elements to ensure that they'll work properly once we add JS functionality.

**Pull request merged**